### PR TITLE
Directoryservice review

### PR DESCRIPTION
**Pull request by Mrt134**
See commit messages for more details. Local build testing reported no issues.

### CODE REVIEW

File: userguide/directoryservice.rst [hunk 1]
```diff
@@ -40,23 +40,22 @@ support within %brand%.  It is strongly recommended to update the
...
 Directory integration.
 
-**Before configuring the Active Directory service**, ensure name
-resolution is properly configured by :command:`ping` ing the domain
-name of the Active Directory domain controller from :ref:`Shell` on
-the %brand% system. If the :command:`ping` fails, check the DNS
-server and default gateway settings in
-:menuselection:`Network --> Global Configuration`
-on the %brand% system.
+Before configuring the Active Directory service, ensure name resolution
```
Comment (wblock) [inline userguide/directoryservice.rst]: This sentence is really, really long and has lots of clauses.  Please shorten or break up.

File: userguide/directoryservice.rst [hunk 2]
```diff
@@ -40,23 +40,22 @@ support within %brand%.  It is strongly recommended to update the
...
+is properly configured by :command:`ping` ing the domain name of the
+Active Directory domain controller from :ref:`Shell` on the %brand%
+system. If the :command:`ping` fails, check the DNS server and default
+gateway settings in :menuselection:`Network --> Global Configuration` on
+the %brand% system.
 
-Next, add a DNS record for the %brand% system on the Windows server
-and verify that the hostname of the %brand% system can be
-pinged from the domain controller.
+Next, add a DNS record for the %brand% system on the Windows server and
```
Comment (wblock) [inline userguide/directoryservice.rst]: Avoid these "Next," constructs.  They usually add little.

File: userguide/directoryservice.rst [hunk 3]
```diff
@@ -40,23 +40,22 @@ support within %brand%.  It is strongly recommended to update the
...
+verify the hostname of the %brand% system can be pinged from the domain
+controller.
 
-Active Directory relies on Kerberos, which is a time sensitive
-protocol. The time on both the %brand% system and the
-Active Directory Domain Controller cannot be out of sync by more than
-a few minutes. The best way to ensure that the same time is running on
-both systems is to configure both systems to:
+Active Directory relies on Kerberos, which is a time sensitive protocol.
```
Comment (wblock) [inline userguide/directoryservice.rst]: s/time sensitive/time-sensitive/

File: userguide/directoryservice.rst [hunk 4]
```diff
@@ -100,133 +99,132 @@ display these settings by checking the box
...
    |                          |               | Mode     |                                                                                                                               |
    +==========================+===============+==========+===============================================================================================================================+
-   | Domain Name              | string        |          | name of Active Directory domain (*example.com*) or child domain (*sales.example.com*); this setting is mandatory and the GUI  |
-   | (DNS/Realm-Name)         |               |          | will refuse to save the settings if the domain controller for the specified domain cannot be found                            |
+   | Domain Name              | string        |          | Name of Active Directory domain (*example.com*) or child domain (*sales.example.com*). This setting is mandatory and the GUI  |
+   | (DNS/Realm-Name)         |               |          | will refuse to save the settings if the domain controller for the specified domain cannot be found.                           |
```
Comment (wblock) [inline userguide/directoryservice.rst]: s/the specified/the/

File: userguide/directoryservice.rst [hunk 5]
```diff
@@ -100,133 +99,132 @@ display these settings by checking the box
...
+   |                          |               |          | slowing down the ability to filter through user and group information.                                                        |
    |                          |               |          |                                                                                                                               |
    +--------------------------+---------------+----------+-------------------------------------------------------------------------------------------------------------------------------+
-   | Use Default Domain       | checkbox      | ✓        | when unchecked, the domain name is prepended to the username; if :guilabel:`Allow Trusted Domains`                            |
-   |                          |               |          | is checked and multiple domains use the same usernames, uncheck this box to prevent name collisions                           |
+   | Use Default Domain       | checkbox      | ✓        | Uncheck to prepend the domain name to the username. If :guilabel:`Allow Trusted Domains`                                      |
```
Comment (wblock) [inline userguide/directoryservice.rst]: Looks like whitespace has been added at the end of these lines.

File: userguide/directoryservice.rst [hunk 6]
```diff
@@ -100,133 +99,132 @@ display these settings by checking the box
...
+   |                          |               |          | is checked and multiple domains use the same usernames, uncheck to prevent name collisions.                                   |
    |                          |               |          |                                                                                                                               |
    +--------------------------+---------------+----------+-------------------------------------------------------------------------------------------------------------------------------+
-   | Allow DNS updates        | checkbox      | ✓        | when unchecked, disables Samba from doing DNS updates when joining a domain                                                   |
+   | Allow DNS updates        | checkbox      | ✓        | Uncheck to disable Samba from doing DNS updates when joining a domain.                                                        |
```
Comment (wblock) [inline userguide/directoryservice.rst]: And this one.

File: userguide/directoryservice.rst [hunk 7]
```diff
@@ -100,133 +99,132 @@ display these settings by checking the box
...
    |                          |               |          |                                                                                                                               |
    +--------------------------+---------------+----------+-------------------------------------------------------------------------------------------------------------------------------+
-   | Domain Controller        | string        | ✓        | will automatically be added to the SRV record for the domain and, when multiple controllers are                               |
-   |                          |               |          | specified, %brand% selects the closest DC which responds; use the short form of the FQDN                                      |
+   | Domain Controller        | string        | ✓        | Will automatically be added to the SRV record for the domain and, when multiple controllers are                               |
+   |                          |               |          | specified, %brand% selects the closest DC which responds. Uses the short form of the FQDN                                     |
```
Comment (wblock) [inline userguide/directoryservice.rst]: This sentence has no period.  The example on the next line should also be a sentence, but without parens.

File: userguide/directoryservice.rst [hunk 8]
```diff
@@ -100,133 +99,132 @@ display these settings by checking the box
...
+   | Site Name                | string        | ✓        | The relative distinguished name of the site object in Active Directory.                                                       |
    |                          |               |          |                                                                                                                               |
    +--------------------------+---------------+----------+-------------------------------------------------------------------------------------------------------------------------------+
-   | Domain Controller        | string        | ✓        | will automatically be added to the SRV record for the domain and, when multiple controllers are                               |
-   |                          |               |          | specified, %brand% selects the closest DC which responds; use the short form of the FQDN                                      |
+   | Domain Controller        | string        | ✓        | Will automatically be added to the SRV record for the domain and, when multiple controllers are                               |
```
Comment (wblock) [inline userguide/directoryservice.rst]: Normally, change "will be" to "is", but in this case it's not needed at all: "Automatically added to..."

File: userguide/directoryservice.rst [hunk 9]
```diff
@@ -301,115 +300,82 @@ automatically once a day as a cron job.
...
-   net ads join -S dcname -U username
-
-
-If no users or groups are listed in the output, these commands can
-provide more troubleshooting information:
-
-.. code-block:: none
-
-   getent passwd
+If no users or groups are listed in the output, :command:`getent passwd`
```
Comment (wblock) [inline userguide/directoryservice.rst]: Flip sentence around to avoid if/then.

File: userguide/directoryservice.rst [hunk 10]
```diff
@@ -301,115 +300,82 @@ automatically once a day as a cron job.
...
+and :command:`getent group` can provide more troubleshooting information.
 
-   getent group
+.. tip:: Sometimes network users will not appear in the drop-down menu
```
Comment (wblock) [inline userguide/directoryservice.rst]: s/will not/do not/

Add a comma before "but"

File: userguide/directoryservice.rst [hunk 11]
```diff
@@ -451,8 +422,7 @@ Next, only run these two commands **if** the
  service sssd start
 
 
-Finally, run these commands. Again, the :command:`echo` command should
-return a *0*:
+Finally, run these commands. The :command:`echo` will return a *0*:
```
Comment (wblock) [inline userguide/directoryservice.rst]: This changes the questionable logic of the original.  It's saying that if "echo $?" does not show a 0, something went wrong.

**Comment by dlavigne** [comment]
Please create a PR for any text improvements in stable.

**Pull request merged**